libyul/backends/wasm/BinaryTransform.cpp

```diff
@@ -311,7 +311,11 @@ bytes BinaryTransform::operator()(BuiltinCall const& _call)
 	bytes args = visit(_call.arguments);
 
 	if (_call.functionName == "unreachable")
-			return toBytes(Opcode::Unreachable);
+		return toBytes(Opcode::Unreachable);
+	else if (_call.functionName == "nop")
```

Comment:
Why expose this? For empty function bodies?

Reply:
The word size transform needs this to avoid having to delete statements.

Reply:
Ah ok. As a placeholder statement `(block)` would also suffice, but `nop` is less invasive.